hack/build/run-functional-tests.sh

```diff
@@ -37,4 +37,13 @@ arg_oc="${KUBECTL:+-oc-path=$KUBECTL}"
 
 test_args="${test_args} -ginkgo.v ${arg_master} ${arg_namespace} ${arg_kubeconfig} ${arg_kubectl} ${arg_oc}"
 
+if [[ $(getClusterType) == $OPENSHIFT_IMAGE ]]; then
+    ./cluster/kubectl.sh adm policy add-scc-to-user privileged -z cdi-sa -n ${CDI_NAMESPACE}
```

Comment:
Let's have a comment explaining why this is needed (and not part of the deployment itself.

Reply:
I should probably do this in the sync instead of the test run.